### ISSUE TITLE
Bicing Providing wrong data

### ISSUE THREAD

**Issue by PMARZV** [eskerda/pybikes#738]
It returns 0 ebikes but has_ebikes returns true
![image](https://github.com/eskerda/pybikes/assets/100836358/700d36f8-6074-406b-b848-3960d921230c)

**Comment by martgnz** [comment]
This happens because AFAIK `has_ebikes` has been used in pybikes to mean that a system/or station is ebike capable, not that there are `ebikes > 0` in a specific station. I agree with you and it makes more sense to flag it true only when there are ebikes in a station, specially in a system where all stations support ebikes.

No Bicing API is being used right now, data is fetched from this JSON file: https://www.bicing.barcelona/get-stations

**Comment by PMARZV** [comment]
Yeah kinda confusing in my opinion, thanks! Should i close the issue or is it better to wait if this possible change is implemented?

**Comment by eskerda** [comment]
The purpose of `has_ebikes` is a bit confusing, but it's meant to indicate if the station supports/supplies electrical bikes. I guess it's a bit redundant, and could be inferred by checking if `ebikes` is present on the extra part. I am thinking maybe to just remove it and avoid confusion
